assets/js/components/Dashboard/PendingScreensPlaceRowAccordion.tsx

```diff
@@ -0,0 +1,164 @@
+  config: ScreenConfiguration;
+}
+
+const formatAppID = (appID: string) => {
+  switch (appID) {
+    // A few labels are presented differently on this page,
+    // to avoid a weird double-":" situation.
+    case "gl_eink":
+    case "gl_eink_v2":
+      return "Green Line E-Ink";
+
+    case "bus_eink":
+    case "bus_eink_v2":
+      return "Bus E-Ink";
+
+    // All other IDs are formatted as usual.
+    default:
+      return SCREEN_TYPES.find(({ ids }) => ids.includes(appID))?.label ?? "";
+  }
+};
+
+// When included in the /configure-screens/* route, app IDs are changed as follows:
+// 1. Remove `_v2` suffix
+// 2. Convert from snake_case to kebab-case
+const appIDAsRoutePart = (appID: string) =>
+  appID.replace("_v2", "").replace("_", "-");
+
+const PendingScreensPlaceRowAccordion: ComponentType<Props> = ({
+  place,
+  appID,
+  placeID,
+  screens,
+  buttonsDisabled,
+  publishCallback,
+}: Props) => {
+  const [hiddenFromScreenplayIDs, setHiddenFromScreenplayIDs] = useState<
+    string[]
+  >([]);
+  const { activeEventKey } = useContext(AccordionContext);
+  const isOpen = activeEventKey?.includes(place.id);
+  const onRowClick = useAccordionButton(place.id);
+  const navigate = useNavigate();
+
+  const handleClickHideCheckbox = (screenID: string) => {
+    if (hiddenFromScreenplayIDs.includes(screenID)) {
+      setHiddenFromScreenplayIDs((prevState) =>
+        prevState.filter((id) => id !== screenID)
+      );
+    } else {
+      setHiddenFromScreenplayIDs((prevState) => [...prevState, screenID]);
+    }
+  };
+
+  const handleClickEdit = () => {
+    navigate(`/configure-screens/${appIDAsRoutePart(appID)}`, {
+      state: { place_id: placeID },
+    });
+  };
+
+  const handleClickPublish = () => {
+    publishCallback(placeID, appID, hiddenFromScreenplayIDs);
+  };
+
+  return (
+    <div
+      className={classNames(
+        "pending-screens-place-row-accordion",
+        isOpen ? "open" : ""
+      )}
+      onClick={onRowClick}
+    >
+      <div className="pending-screens-place-row-accordion__header">
+        <Container fluid>
+          <Row className="align-items-center text-white">
+            <Col
+              lg={5}
+              className="place-info-col d-flex align-items-center justify-content-start"
+            >
+              <AccordionToggle eventKey={place.id} hidden={false} />
+              <div className="place-name">
+                {capitalizeTerminalStops(place.name, null)}
+              </div>
+              <div className="place-id">Station ID: {place.id}</div>
+            </Col>
+            <Col className="d-flex justify-content-center">
+              Type: {formatAppID(appID)}
+            </Col>
+            <Col className="buttons d-flex justify-content-end">
+              <Button
+                className="edit-button"
+                onClick={handleClickEdit}
+                disabled={buttonsDisabled}
+              >
+                <PencilSquare /> Edit Pending
```

Comment:
I'm seeing some slight misalignment in this button. Should just need a slight pixel shift.
![Screenshot 2024-03-21 at 1 56 37 PM](https://github.com/mbta/screenplay/assets/10713153/c64aa5d6-fdc7-4528-b705-33baaa56291d)